devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -205,17 +201,18 @@ def configure() {
 	// enrolls with default periodic reporting until newer 5 min interval is confirmed
 	sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 
-	// OnOff minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
+	// OnOff, level minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
+	zigbee.onOffConfig(0, 300) +
```

Comment:
@KKlimczukS on re-review, I suggest doing the configs last like they were done in the original code. That way the current values are read first which is what the user cares about the most. So it should be:
* set color temperature
* refresh
* configure reporting

Reply:
> @KKlimczukS on re-review, I suggest doing the configs last like they were done in the original code. That way the current values are read first which is what the user cares about the most. So it should be:
> 
> * set color temperature
> * refresh
> * configure reporting

@tpmanley ok, done.